### PR TITLE
TM-8601 Provide documentation for data-disable-delay attribute (16.0)

### PR DESCRIPTION
**Pull request by musialp**
Provide documentation for data-disable-delay attribute (skip link tracking)

### CODE REVIEW

File: tag_manager/skip_link_tracking.rst [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+Introduction
+------------
+As per the `MDN`_ definition:
+
+    The <a> HTML element (or anchor element), with its href attribute, creates
+    a hyperlink to web pages, files, email addresses, locations in the same page,
+    or anything else a URL can address.
+
+If you wish to trigger tags, when the anchor element is clicked, they need time
+to execute before the redirect happens. That is why, our container is equiped
```
Comment (zawias-pro) [inline tag_manager/skip_link_tracking.rst:20]:
```suggestion
to execute before the redirect happens. That is why our container is equipped
```

File: tag_manager/skip_link_tracking.rst [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+However, not every anchor element is supposed to perform a redirect. One such
+example can be SPA pages where ``a`` elements can serve as buttons. In such case,
+the action performed inside the container can break the functionality of the page.
+That is where the ``data-disable-delay`` attribute comes in.
+
+`data-disable-delay` attribute
+------------------------------
+``data-disable-dalay`` is special custom attribute that is recognized by the
+container. Once the anchor element is clicked and the aformentioned attribute
```
Comment (zawias-pro) [inline tag_manager/skip_link_tracking.rst:41]:
```suggestion
container. Once the anchor element is clicked and the aforementioned attribute
```

File: tag_manager/skip_link_tracking.rst [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+is detected on the element, it tells the container to skip the execution of the
+logic responsible for delaying the default action. Listeners attached to the
+element are executed immediatly after clicking.
+
+
+Example
+```````
+
+#.  Lets assume that your Tag Manager setup includes a `Custom code (async)` tag
```
Comment (zawias-pro) [inline tag_manager/skip_link_tracking.rst:50]:
```suggestion
#.  Let's assume that your Tag Manager setup includes a `Custom code (async)` tag
```

File: tag_manager/skip_link_tracking.rst [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+    (the contents of the tag does not matter in this case) and a basic `Click
+    trigger` assigned to said tag
```
Comment (zawias-pro) [inline tag_manager/skip_link_tracking.rst:52]:
```suggestion
    trigger` assigned to the said tag.
```

**Review by zawias-piwik-pro — CHANGES_REQUESTED** [review]

File: tag_manager/skip_link_tracking.rst (hunk 1 above)
Comment (musialp) [reply tag_manager/skip_link_tracking.rst]: Done

File: tag_manager/skip_link_tracking.rst (hunk 2 above)
Comment (musialp) [reply tag_manager/skip_link_tracking.rst]: Done

File: tag_manager/skip_link_tracking.rst (hunk 3 above)
Comment (musialp) [reply tag_manager/skip_link_tracking.rst]: Done

File: tag_manager/skip_link_tracking.rst (hunk 4 above)
Comment (musialp) [reply tag_manager/skip_link_tracking.rst]: Done

**Review by zawias-piwik-pro — APPROVED** [review]

**Review by kororokke — APPROVED** [review]

**Pull request merged**